### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by LaurensKrudde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+#### Docker
+
+Make sure you have Docker and Docker Compose installed.
+
+#### Clone
+
+Clone the operation repository:
+```
+git clone https://github.com/remla23-team15/model-service.git
```
Comment (MattiaBonfanti-CS) [inline README.md]: Small typo:

 https://github.com/remla23-team15/operation.git

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    image: "ghcr.io/remla23-team15/app:latest"
+    ports:
+      - 8080:80
+    depends_on:
+      - model-service
+
+  model-service:
+    image: "ghcr.io/remla23-team15/model-service:latest"
+    ports:
+      - 8081:80
```
Comment (MattiaBonfanti-CS) [inline docker-compose.yml]: `model-service` exposes port 8080, and the `app` looks for it on that port so this line should become:

- 8080:8080

**Review by MattiaBonfanti-CS — APPROVED** [review]
Very good overall, just added some small comments. 

Well done!

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# docker-compose.yml
+services:
+
+  app:
+    image: "ghcr.io/remla23-team15/app:latest"
+    ports:
+      - 8080:80
```
Comment (MattiaBonfanti-CS) [inline docker-compose.yml]: See comment below, `app` looks for `model-service` on port 8080, so this line should become:

- 8081:80

**Review by MattiaBonfanti-CS — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (LaurensKrudde) [reply docker-compose.yml]: I see, thanks!

**Pull request merged**